app/models/classification.rb

```diff
@@ -377,12 +377,8 @@ def self.export_to_yaml
   def export_to_array
     h = attributes
     h["name"] = name
-    if category?
-      ["id", "tag_id", "reserved"].each { |k| h.delete(k) }
-      h["entries"] = entries.collect(&:export_to_array).flatten
-    else
-      ["id", "tag_id", "reserved", "parent_id"].each { |k| h.delete(k) }
-    end
+    %w(id tag_id reserved parent_id).each { |k| h.delete(k) }
```

Comment:
Perhaps `h = h.except(*%w(id tag_id reserved parent_id))`?  I feel like we shouldn't modify the attributes hash unless we are sure it's a copy.